content/references/rippled-api/transaction-formats/transaction-metadata.md

```diff
@@ -6,25 +6,75 @@ Transaction metadata is a section of data that gets added to a transaction after
 
 Some fields that may appear in transaction metadata include:
 
-| Field                                 | Value               | Description    |
-|:--------------------------------------|:--------------------|:---------------|
-| `AffectedNodes`                       | Array               | List of [ledger objects](ledger-object-types.html) that were created, deleted, or modified by this transaction, and specific changes to each. |
-| `DeliveredAmount`                     | [Currency Amount][] | **DEPRECATED.** Replaced by `delivered_amount`. Omitted if not a partial payment. |
-| `TransactionIndex`                    | Unsigned Integer    | The transaction's position within the ledger that included it. This is zero-indexed. (For example, the value `2` means it was the 3nd transaction in that ledger.) |
-| `TransactionResult`                   | String              | A [result code](transaction-results.html) indicating whether the transaction succeeded or how it failed. |
-| [`delivered_amount`](#delivered-amount) | [Currency Amount][] | The [Currency Amount][] actually received by the `Destination` account. Use this field to determine how much was delivered, regardless of whether the transaction is a [partial payment](partial-payments.html). [New in: rippled 0.27.0][] |
+{% include '_snippets/tx-metadata-field-table.md' %} <!--_ -->
+
+## Example Metadata
+
+The following JSON object shows the metadata for [a complex cross-currency payment](https://xrpcharts.ripple.com/#/transactions/8C55AFC2A2AA42B5CE624AEECDB3ACFDD1E5379D4E5BF74A8460C5E97EF8706B):
+
+```json
+{% include '_code-samples/metadata/cross-currency-payment.json' %}
+```
+
+## AffectedNodes
+
+The `AffectedNodes` array contains a complete list of the [objects in the ledger](ledger-object-types.html) that this transaction modified in some way. Each entry in this array is an object with one top-level field indicating what type it is:
+
+- `CreatedNode` indicates that the transaction created a new object in the ledger.
+- `DeletedNode` indicates that the transaction removed an object from the ledger.
+- `ModifiedNode` indicates that the transaction modified an existing object in the ledger.
+
+The value of each of these fields is an object describing the changes made to the object.
```

Comment:
The two instances of "object" make this a little confusing. Is the second one the "object in the ledger"?

Reply:
Yup. Changed to:

> The value of each of these fields is a JSON object describing the changes made to the ledger object.